src/engine/cachingreader.cpp

```diff
@@ -89,7 +89,7 @@ void CachingReader::freeChunk(CachingReaderChunkForOwner* pChunk) {
 }
 
 void CachingReader::freeAllChunks() {
-    for (CachingReaderChunkForOwner* pChunk: m_chunks) {
+    for (CachingReaderChunkForOwner* pChunk: qAsConst(m_chunks)) {
```

Comment:
Not sure here, but normally I would use here following the for(:) patterns.  

for (const auto& pChunk: qAsConst(m_chunks))

I am unsure about the implications of a reference to a pointer.

Reply:
see: 
src/engine/enginebuffer.cpp:1280

Reply:
Yea, this was just a robotic change of only the right-hand side to avoid the potential detach of `m_chunks`. Since the pointer is trivially copyable I think there's no downside to `Foo*` (a copy) vs. `const auto&` (a reference to Foo*).

Reply:
changed to `const auto&`